crates/driver/src/commit_reveal.rs

```diff
@@ -7,8 +9,12 @@ use solver::{
 
 /// A `SolutionSummary` holds all information solvers are willing to disclose during settlement
 /// competition. It does **not** have to include the call data, yet.
-#[derive(Debug, Clone, Deserialize, Serialize)]
-pub struct SettlementSummary {}
+#[derive(Debug, Default, Clone, Deserialize, Serialize)]
+pub struct SettlementSummary {
```

Comment:
naming-nit: I would call this a "proposal", since it is the settlement that you are proposing to solve.

Reply:
*Edit*: I see that you use "proposal" for the internal settlement type of the driver and not what is sent to the `autopilot`.

This is also consistent with the `InteractionProposal` naming, and I like it as is now.